fintual-oss/paraffin-front
# Resource section without authentication

# Context

With this PR, we are modifying the components in the `ResourceSection` of the `learning-units/[id]` page (in the sidebar) to display them when a user is not authenticated. In particular, we modified the `AddEvaluation` component, which allows a user to make an evaluation. 

With the introduced changes, the widgets in `AddEvaluation` are disabled, so they don't trigger any action. Also, a tooltip is displayed when the mouse is passed over and a modal is displayed when clicked on the evaluation card.

# Changelog
* The content of `AddEvaluation` was moved to `AddEvaluationAuth` component.
* Create component `AddEvaluationNoAuth`
* Functions which needed user authentication were moved from `ResourceSection` to `AddEvaluationAuth`

# QA
1. Log out
2. go to http://localhost:3001/learning-units/1
3. In any resource, click on the "Ver recurso" button.
4. In the sidebar, the widgets to evaluate the resource should (Text area, rating, buttons) be disabled.
5.  Pass the mouse over the widgets, the message "Ingresa para poder evaluar un recurso" is displayed.
6. Click on the evaluation card, a modal inviting to sign in is displayed.
